examples/tectonic.libvirt.yaml

```diff
@@ -11,7 +11,7 @@ admin:
 baseDomain:
 
 libvirt:
-  uri: qemu:///system
+  uri: qemu+tcp://192.168.122.1/system
```

Comment:
Do we need this?  If so, can you talk about it in either a comment here or in `Documentation/dev/libvirt-howto.md`?  I don't understand 192.168.122.1 vs. the 192.168.124.0/24 you have in 15f2f456d2.

Reply:
This just needs to be an IP on the host that libvirt is listening on, and that the actuator pod can connect to from within the cluster. Most distros seem to default to `192.168.122.1` for the default `virbr0` interface. You can't use something from the `192.168.124.0/24` subnet if that's what you've configured for the Tectonic network, as it won't exist yet.

Would you prefer I just add a node and leave this for the user to configure?

Reply:
Fixed. If this isn't enough, please let me know what you'd like here.

Reply:
Your new comment in 3d5422b5a3 works for me, thanks :).